Mgrate to null safety

techno-disaster opened the pull request:
Support null safety and Fixes #10

techno-disaster committed:
migrate to null safety

Leptopoda commented:
Hi there,
Thanks for the work,
You should probably also update the example shown in the readme files (in the example and the root)

lilmistake committed:
Update navigation_rail.dart

Fixed overflow problem by making nav rail scrollable

lilmistake committed:
Dynamic sizing of widget

Added intrinsic height widget

lilmistake committed:
Update navigation_rail.dart

Expanded nav destination vertically and added a padding to overall navrail.

lilmistake committed:
Conditionally expanding rail to max width

Conditionally expanding rail to max width, expanding for medium screen size causes unbounded width constraint error

techno-disaster committed:
fix: overflow problem by making nav rail scrollable

NandaSBinod committed:
updated deprecated accentColor

NandaSBinod committed:
updated accentColor in example

techno-disaster committed:
Merge pull request #4 from NandaSBinod/master

Migrated accentColor to colorScheme.secondary
